lib/commands/jukebox/Jukebox.js

```diff
@@ -11,10 +11,11 @@ class Jukebox {
         this.queue = []
         this.nowPlaying = null
         this.playlist = null
+        this.isReplaying = false
 
         this.start = this.start.bind(this)
         this.stop = this.stop.bind(this)
-        this.next = this.next.bind(this)
+        this.next = this.nextOne.bind(this)
```

Comment:
The names here should be consistent.Both next and nextOne should be themselves bound to this. Right now next is bound to nextOne, so there would actually be an infinite loop if you check the definition of nextOne.